packages/legend-studio/src/stores/ApplicationConfig.ts

```diff
@@ -141,7 +131,7 @@ class ApplicationCoreOptions {
   ): ApplicationCoreOptions {
     const config = ApplicationCoreOptions.serialization.fromJson(configData);
     if (config.DEV__enableStrictRuntimeChecks) {
```

Comment:
do we even need this config `DEV__enableStrictRuntimeChecks`

Reply:
We don't but will leave it here for now, we will need it when we do https://github.com/finos/legend-studio/issues/288 I think